This script is needed to patch the flutter.js and flutter_service_worker.js so they load faster 
see: https://github.com/dwyl/app/issues/326#issuecomment-1478314967

--- flutter.js	2023-03-24 18:27:43
+++ flutter2.js	2023-03-24 18:37:08
@@ -353,7 +353,15 @@
       // Install the `didCreateEngineInitializer` listener where Flutter web expects it to be.
       this.didCreateEngineInitializer =
         entrypointLoader.didCreateEngineInitializer.bind(entrypointLoader);
-      return entrypointLoader.loadEntrypoint(entrypoint);
+      
+        const main = entrypointLoader.loadEntrypoint(entrypoint);
+
+        fetch("assets/FontManifest.json");
+        fetch("assets/fonts/MaterialIcons-Regular.otf");
+        fetch("assets/packages/cupertino_icons/assets/CupertinoIcons.ttf");
+        fetch("/");
+
+        return main;
     }
   }